openquakeplatform_building_class/migrations/0001_initial.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from django.conf import settings
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ClassificationHead',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('country', models.CharField(max_length=3)),
```

Comment:
`country` is fine, but perhaps `country_iso3` would be more explicit

Reply:
the migration is made with the current models, not new things.